### PR TITLE
Merging dev to master

### PR DESCRIPTION
**Pull request by arisaves**
Premature but I wanted WAVE errors covered sooner than later

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import React from 'react';
+import ReactGA from 'react-ga';
+ReactGA.initialize('UA-134781234-1');
```
Comment (ghost) [inline src/App.js:3]: space after imports

File: src/stylesheets/components/_footer.scss [hunk 2]
```diff
@@ -4,13 +4,13 @@
     list-style-type: none;
     margin-bottom: 1em;
 
-    @media (min-width: $breakpoint-small) {
+    @media (min-width: $breakpoint-medium) {
```
Comment (ghost) [inline src/stylesheets/components/_footer.scss:7]: are these from uikit? I should start using these more...

**Review by ghost — APPROVED** [review]
This looks good to me :shipit: 🚀 !

File: src/stylesheets/components/_footer.scss (hunk 2 above)
Comment (arisaves) [reply src/stylesheets/components/_footer.scss:7]: yes! these are supplied by uikit, they are super handy.

**Pull request merged**